README.md

```diff
@@ -669,6 +669,7 @@ Services to securely store your Docker images.
 - [Security Best Practices for Building Docker Images](https://linux-audit.com/tag/docker/)
 - [Software Engineering Radio interview of Docker Security Team Lead (Diogo Mónica)](http://www.se-radio.net/2017/05/se-radio-episode-290-diogo-monica-on-docker-security/)
 - [Top ten most popular docker images each contain at least 30 vulnerabilities](https://snyk.io/blog/top-ten-most-popular-docker-images-each-contain-at-least-30-vulnerabilities/)
+- [Ten Docker Image Security Best Practices Cheat Sheet](https://snyk.io/blog/10-docker-image-security-best-practices)
```

Comment:
There is a redirect from https://snyk.io/blog/10-docker-image-security-best-practices => https://snyk.io/blog/10-docker-image-security-best-practices/

Also, all the entries are sorted alphabetically. It should be one up.

Reply:
Ugh, missed again on the alphabetical ordering. Sorry :)